src/runtime_src/core/pcie/tools/cloud-daemon/msd.cpp

```diff
@@ -44,48 +45,47 @@ bool quit = false;
 uint64_t chanSwitch = (1UL<<MAILBOX_REQ_TEST_READY) |
                       (1UL<<MAILBOX_REQ_TEST_READ)  |
                       (1UL<<MAILBOX_REQ_LOAD_XCLBIN);
-
-// Suppport for msd plugin
+// Support for msd plugin
 void *plugin_handle;
 init_fn plugin_init;
 fini_fn plugin_fini;
 struct msd_plugin_callbacks plugin_cbs;
-static const std::string plugin_path("/lib/firmware/xilinx/msd_plugin.so");
+static const std::string plugin_path("/opt/xilinx/xrt/lib/libmsd_plugin.so");
 
 // Init plugin callbacks
 static void init_plugin()
 {
-    plugin_handle = dlopen(plugin_path.c_str(), RTLD_LAZY | RTLD_GLOBAL);
-    if (plugin_handle == nullptr)
-        return;
-
-    syslog(LOG_INFO, "found msd plugin: %s", plugin_path.c_str());
-    plugin_init = (init_fn) dlsym(plugin_handle, INIT_FN_NAME);
-    plugin_fini = (fini_fn) dlsym(plugin_handle, FINI_FN_NAME);
-    if (plugin_init == nullptr || plugin_fini == nullptr)
-        syslog(LOG_ERR, "failed to find init/fini symbols in plugin");
+	plugin_handle = dlopen(plugin_path.c_str(), RTLD_LAZY | RTLD_GLOBAL);
```

Comment:
why make indentation change in this file? In user land code, i have been using 4-space indentation for all daemon files...

Reply:
will check and change all files